### ISSUE TITLE
Feature request: Add ability to create Azure Storage Table

### ISSUE THREAD

**Issue by bsnchan** [hashicorp/terraform#7257]
It would be nice to have the ability to create storage tables using the azurerm terraform resource.

Something along the lines of:

```hcl
resource "azurerm_storage_table" "test" {
    name = "test"
    resource_group_name = "${azurerm_resource_group.resourcegroup.name}"
    storage_account_name = "${azurerm_storage_account.storageaccount.name}"
}
```

**Comment by stack72** [comment]
Hi @bsnchan 

Do you mean Azure Table Storage? I can't find anything in the Azure documentation for Storage Table

Paul

**Comment by stack72** [comment]
Hi @bsnchan 

A PR is currently waiting to be merged for this :) It should be in 0.7

Paul

**Comment by bsnchan** [comment]
@stack72 perfect! Thanks so much for the quick fix!
